client/components/tinymce/plugins/advanced/plugin.jsx

```diff
@@ -53,10 +56,16 @@ function advanced( editor ) {
 	editor.addButton( 'wpcom_advanced', {
 		text: 'Toggle Advanced',
 		tooltip: 'Toggle Advanced',
-		classes: 'btn advanced',
+		classes: 'btn wpcom-icon-button advanced',
 		cmd: 'WPCOM_ToggleAdvancedVisible',
 		onPostRender: function() {
 			button = this;
```

Comment:
"button" is getting used in 3 different ways in this function;  It'd be nice to have some help understanding what's going on.  The clearest thing might be to rename this button closure to something like `advancedButton`.  I'm tempted to suggest a comment explaining that we're capturing it here for use in `updateVisibility()`, but I think that will be a lot clearer if it's got a unique name.